### DISCUSSION TITLE
Is there a way to always show the scrollbars?

### DISCUSSION THREAD

**Discussion by Mahibvk** [tailwindlabs/tailwindcss#8890] [Help]
I want the scrollbar to be always visible like in the picture from the documentation, What property is required to show the scrollbar when it's not focused and not scrolled? 
I couldn't find anything in the docs tbh or I missed something.

```
    <div className="bg-white shadow sm:rounded-lg mt-8 lg:w-1/2 ml-3 h-96 overflow-scroll">Main Content</div>
```
<img width="843" alt="Screenshot 2022-07-19 at 12 46 49 PM" src="https://user-images.githubusercontent.com/100900653/179689877-44c923ff-7f21-414e-b255-1fec5e84d027.png">

**Comment by ichenglin** [comment]


**Reply by Mahibvk** [reply]
@ichenglin Thanks for the answer, I added an other div inside the parent as you've mentioned, it didn't seem to work.

**Comment by Mahibvk** [comment]
It was a typo, sorry about that. The answer you gave didn't seem to work

*B V K Mahijendra*

+917330760052

Front-End Developer

https://nium.org.in

https://tsbpass.telangana.gov.in/


<https://tsbpass.telangana.gov.in/> <https://nium.org.in/>


<https://www.facebook.com/Telangana-State-Building-Permission-Approval-Self-Certification-System-106603121903162/?ref=pages_you_manage>
<https://www.instagram.com/ts_bpass/?hl=en> <https://twitter.com/TS_bPASS>
<https://www.linkedin.com/company/niumhyd/>
